models.py

```diff
@@ -487,20 +491,22 @@ def __call__(self, args, data):
         if len(args) < 2:
             return "Not enough arguments."
         elif len(args) == 2:
-            match = re.match(r'@?([A-Za-z0-9]{,32})', args[1])
-            if match is None:
+            user = re.match(r'@?([A-Za-z0-9]{,32})', args[1])
```

Comment:
Should update to use the actual Beam username regex - this won't catch `Alkali_Metal`, for example (no underscore).
```python
re.match(r'@?([\w_-]*[a-z][\w_-]*)', args[1], re.I)
```